### PR TITLE
Start implementing code to run the analysis.

### PR DESCRIPTION
**Pull request by garretrieger**

### CODE REVIEW

File: analysis/analyzer.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""Main binary for the analyis framework.
+
+Takes an input data file and tests various progressive font enrichment methods
+against each one.
+
+Collects an overall score for each method and reports the aggregate results.
```
Comment (rsheeter) [inline analysis/analyzer.py:6]: clarify what a "score" is or link to where clarified

File: analysis/sample/sample_data_set.textproto [hunk 2]
```diff
@@ -0,0 +1,30 @@
+sequences: {
+  page_views: {
+    contents: {
+      font_name: "Roboto-Regular.ttf"
```
Comment (rsheeter) [inline analysis/sample/sample_data_set.textproto:4]: future consideration: it's quite plausible to encounter multiple different versions of a file. Maybe we need url of font or even a checksum?

**Review by rsheeter — APPROVED** [review]

File: analysis/sample/sample_data_set.textproto (hunk 2 above)
Comment (garretrieger) [reply analysis/sample/sample_data_set.textproto]: Will be fixed in an up comming PR.

Comment (garretrieger) [reply analysis/sample/sample_data_set.textproto:4]: The intent is that sort of concern is handled during the preparation of the input data set.  The input data set is both the set of font files and the page view sequences. If there's multiple versions of the same font then during input data set preparation the different versions should be fetched and given unique names and then the page view sequences should refer to the different versions by the appropriate names.

**Pull request merged**